edge-modules/MetricsCollector/Program.cs

```diff
@@ -29,18 +32,20 @@ private static async Task<int> MainAsync()
             try
             {
                 moduleClient = await ModuleClient.CreateFromEnvironmentAsync(transportSettings);
+                Dictionary<string, string> testSpecificTags = await GetTestSpecificTagsFromTwin(moduleClient);
+
                 MetricsScraper scraper = new MetricsScraper(Settings.Current.Endpoints);
                 IMetricsPublisher publisher;
                 if (Settings.Current.UploadTarget == UploadTarget.AzureLogAnalytics)
                 {
-                    publisher = new LogAnalyticsUpload(Settings.Current.AzMonWorkspaceId, Settings.Current.AzMonWorkspaceKey, Settings.Current.AzMonLogType);
+                    publisher = new LogAnalyticsUpload(Settings.Current.LogAnalyticsWorkspaceId, Settings.Current.LogAnalyticsWorkspaceKey, Settings.Current.LogAnalyticsLogType);
                 }
                 else
                 {
                     publisher = new EventHubMetricsUpload(moduleClient);
                 }
 
-                using (MetricsScrapeAndUpload metricsScrapeAndUpload = new MetricsScrapeAndUpload(scraper, publisher, Guid.NewGuid()))
+                using (MetricsScrapeAndUpload metricsScrapeAndUpload = new MetricsScrapeAndUpload(scraper, publisher, testSpecificTags))

```

Comment:
nit:use var
```suggestion
                using (var metricsScrapeAndUpload = new MetricsScrapeAndUpload(scraper, publisher, testSpecificTags))
```

Reply:
I prefer not to use var. Is this a strict rule?